### ISSUE TITLE
Certain schedules might break the API

### ISSUE THREAD

**Issue by janaurka** [starkandwayne/shield#391]
Hey,

I’m running:
* Shield 8.0.6
* Deployed through the `bosh-release`
* on vSphere 5.5/6.0

Schedule changes via the `cli` only work with something like `daily 2am`. 

If I do stuff like `"every 2h at 11am"` it fails to parse the json with something like: 

```
/var/vcap/packages/shield/bin/shieldd: ERROR: GET /v2/tenants/<tenant-uuid>/systems errored: parsing failed
```
That’s from the `shieldd` error log in `INFO` mode.

This breaks the API and therefore the WebUI which will not display any jobs anymore.

Interestingly the `cli` is displaying it if it would not be a problem.

It looks like that `daily at <something` works fine. But I did not manage to get a proper `every <something> at <something else>` running

Expected result would be:
* Invalid entries are being refused by the API
* The WebUI should still be able to render (at least other jobs)
* Every schedule should be possible

**Comment by jhunt** [comment]
The expectations are spot-on, and we will make it so.

**Comment by jhunt** [comment]
Reopening this until it gets released.

**Comment by jhunt** [comment]
This was released in 8.0.8; @janaurka can you confirm that this part is working for you, even if the help is a bit misleading?

**Comment by janaurka** [comment]
all the things i’ve tested so far worked! guess we can close this. i’d open it again if i would hit an edgecase, but for now i’m happy. thx for the fix!
